docs/modules/ROOT/pages/system-properties.adoc

```diff
@@ -569,6 +569,14 @@ idle CPU usage but may result in decreased latency. Higher values will increase
 |int
 |Some JMX MBeans are cached  to reduce the overhead of calculating statistics. This parameter determines for how long the MBeans can go stale.
 
+|`hazelcast.journal.cleanup.threshold`
+| 1.0f
```

Comment:
the `f` suffix is not necessary when configuring the property and may be puzzling for for non-java users